backend/main.py

```
@@ -2,36 +2,39 @@
 from fastapi.responses import JSONResponse
 from loguru import logger
 
+from backend.candidate_comparisons.router import (
+    router as candidate_comparison_router,
```

Ruff (0.12.2)

`backend.candidate_comparisons.router.router` imported but unused

Remove unused import: `backend.candidate_comparisons.router.router`

(F401)